integration_test/integration_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Fail on any error.
+set -eo pipefail
+
+# Display commands being run.
+set -x
+
+SERVICE_KEY="${KOKORO_KEYSTORE_DIR}/72935_cloud-profiler-e2e-service-account-key"
+COMMIT=$(git rev-parse HEAD)
```

Comment:
Not sure if you tried this script. So for Github, Kokoro checks out the source code into the current directory? I remember we need to cd into git/{repository-name} for Git on Borg.

Reply:
Thanks, I haven't run this yet and you're totally correct. Looks like I need to cd into github/{repository-name} for github.
Done.

Reply:
Below should now "cp -R integration_test $GOPATH/src"?

Reply:
Yes, fixed.